### ISSUE TITLE
how to understand  tile_size_height and tile_stride_height args in  wan training script

### ISSUE THREAD

**Issue by Feynman1999** [modelscope/DiffSynth-Studio#506]
parser.add_argument(
        "--tile_size_height",
        type=int,
        default=34,
        help="Tile size (height) in VAE.",
    )
    parser.add_argument(
        "--tile_size_width",
        type=int,
        default=34,
        help="Tile size (width) in VAE.",
    )
    parser.add_argument(
        "--tile_stride_height",
        type=int,
        default=18,
        help="Tile stride (height) in VAE.",
    )
    parser.add_argument(
        "--tile_stride_width",
        type=int,
        default=16,
        help="Tile stride (width) in VAE.",
    )

Is this size referring to the number of RGB pixels or the latent space?

**Comment by Artiprocher** [comment]
@Feynman1999 It is referring to the latent space. These parameters are not sensitive to the result. 

`tile_size_height=34, tile_size_width=34` corresponds to 272*272 pixels.

If your GPU memory is limited, we recommend enabling tile to reduce memory usage; if your GPU memory is sufficient, we recommend disabling tile to improve speed.

**Comment by Feynman1999** [comment]
> [@Feynman1999](https://github.com/Feynman1999) It is referring to the latent space. These parameters are not sensitive to the result.
> 
> `tile_size_height=34, tile_size_width=34` corresponds to 272*272 pixels.
> 
> If your GPU memory is limited, we recommend enabling tile to reduce memory usage; if your GPU memory is sufficient, we recommend disabling tile to improve speed.

Thank you for your explanation. I would like to ask if the tile operation can be used to reduce the memory usage during VAE training (for example, some tasks require mapping the latent space back to RGB space to calculate the loss)
